corehq/apps/locations/permissions.py

```diff
@@ -20,11 +20,36 @@ def is_locations_admin(view_fn):
     return locations_access_required(domain_admin_required(view_fn))
 
 
-def user_can_edit_location(user, location):
-    user_loc = user.get_location(location.domain).sql_location
+def user_can_edit_location(user, location, project):
+    """
+    Expects SQLLocation
```

Comment:
in this instance I think rather than add docs it'd be better to rename the variable to `sql_location`